src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -94,7 +94,9 @@ public async Task TestOpenSolution_MultiProjectSolution()
             }
         }
 
-        [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/41456")]
+        // When this test is unskipped, the following should be used instead of the Fact attribute.
+        // [ConditionalFact(typeof(VisualStudioMSBuildInstalled)), Trait(Traits.Feature, Traits.Features.MSBuildWorkspace)]
```

Comment:
Note that the 'AlwaysSkip' property on ConditionalFact probably would have the desired effect here.